FirebaseRemoteConfig/Sources/RCNConfigContent.h

```diff
@@ -65,8 +67,8 @@ typedef NS_ENUM(NSInteger, RCNDBSource) {
 /// Gets the active config and Personalization metadata.
 - (NSDictionary *)getConfigAndMetadataForNamespace:(NSString *)FIRNamespace;
 
-/// Sets the fetched rollout metadata to active and return the active rollout metadata.
-- (NSArray<NSDictionary *> *)activateRolloutMetadata;
+/// Sets the fetched rollout metadata to active.
+- (void)activateRolloutMetadata;
```

Comment:
Usually even we don't return the results, we want to return `BOOL` to indicate if the active is successful or not, wonder if we need to do it here?

Reply:
I followed the personalization implementation of activation.
If this method fails, the activeRolloutMetadata will still be the same one as previous and send to Crashlytics.
I was wondering what should we do if this method failed.

Reply:
Add a success completion handler to the method, only notify Crashlytics when success is true.